crates/fluvio-smartengine/src/transforms/simple_transform.rs

```diff
@@ -0,0 +1,83 @@
+use fluvio_smartmodule::dataplane::smartmodule::{
+    SmartModuleInput, SmartModuleOutput, SmartModuleTransformErrorStatus,
+};
+use wasmtime::{TypedFunc, AsContextMut};
+use anyhow::Result;
+
+use crate::{
+    instance::{SmartModuleInstanceContext, SmartModuleTransform},
+    state::WasmState,
+};
+
+type WasmFn = TypedFunc<(i32, i32, u32), i32>;
+
+pub const FILTER_FN_NAME: &str = "filter";
```

Comment:
change to pub(crate).  They don't need to be exported

Reply:
`pub(crate)` is unnecessary since `simple_transform` is a private mod.

Reply:
But I can change to that if you still prefer that style :)

Reply:
make it more proof against future changes